package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@haventec/common-web-sdk",
+  "version": "1.0.0",
+  "description": "Haventec Common Web SDK",
+  "keywords": [],
+  "author": "Haventec <support@haventec.com>",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Haventec/common-web-sdk.git"
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "prepare": "tsc",
+    "test": "./node_modules/karma/bin/karma start karma.conf.js"
```

Comment:
Can you add runners for building and pre-publish, so that whatever needs to be run for building gets run - transpiling, testing, coverage, auditing, packaging, etc. The "prepare" script entry is good for running everything required for building/publishing

Reply:
"test" does all of that. 
"prepare" runs what you need before publishing